// heroku-cli-util/index.js
'use strict'

exports.color = require('heroku-cli-util/node_modules/@heroku-cli/color').default

var console = require('heroku-cli-util/lib/console')
var errors = require('heroku-cli-util/lib/errors')
var prompt = require('heroku-cli-util/lib/prompt')
var styled = require('heroku-cli-util/lib/styled')

exports.hush = console.hush
exports.log = console.log.bind(console)
exports.formatDate = require('heroku-cli-util/lib/date').formatDate
exports.error = errors.error
exports.action = require('heroku-cli-util/lib/action')
exports.warn = exports.action.warn
exports.errorHandler = errors.errorHandler
exports.console = console
exports.yubikey = require('heroku-cli-util/lib/yubikey')
exports.prompt = prompt.prompt
exports.confirmApp = prompt.confirmApp
exports.preauth = require('heroku-cli-util/lib/preauth')
exports.command = require('heroku-cli-util/lib/command')
exports.debug = console.debug
exports.mockConsole = console.mock
exports.table = require('heroku-cli-util/lib/table')
exports.stdout = ''
exports.stderr = ''
exports.styledHeader = styled.styledHeader
exports.styledObject = styled.styledObject
exports.styledHash = styled.styledObject
exports.styledNameValues = styled.styledNameValues
exports.styledJSON = styled.styledJSON
exports.open = require('heroku-cli-util/lib/open')
exports.got = require('heroku-cli-util/lib/got')
exports.linewrap = require('heroku-cli-util/lib/linewrap')
exports.Spinner = require('heroku-cli-util/lib/spinner')
exports.exit = require('heroku-cli-util/lib/exit').exit
exports.auth = require('heroku-cli-util/lib/auth')
exports.login = exports.auth.login
exports.logout = exports.auth.logout
